components/stepper/themes/express.css

```diff
@@ -11,14 +11,23 @@ governing permissions and limitations under the License.
 */
 
 @container (--system: express) {
-  /* @todo remove this when stepper is migrated; this ensures stepper buttons inherit border color and width */
   .spectrum-Stepper {
-    --spectrum-stepper-border-width: var(--spectrum-border-width-200); /* express stepper has border */
+    --spectrum-stepper-border-width: var(--spectrum-border-width-200);
+
+    --spectrum-stepper-buttons-border-style: solid;
+    --spectrum-stepper-buttons-border-width: var(--spectrum-border-width-200);
+    --spectrum-stepper-buttons-background-color: var(--spectrum-gray-50);
+
+    --spectrum-stepper-button-border-width: 0;
```

Comment:
On this line, `-button-` is singular in the name, but in `themes/spectrum.css` it looks like it's plural `-buttons-` and I think `-buttons` is what we refer to in the `index.css`.

Can you take a look to make sure that the naming is correct?

Reply:
You're right! It doesn't look like that variable is being used anymore (along with a few others) now that I'm using in-field button so I've removed them